### PR TITLE
Fix typos and mistranslations of Ethereum

### PR DESCRIPTION
**Pull request by lukassim**
## Description

Fix common typo in the spelling of Ethereum, specifically 'Etherum'.
This is the most common typo in the spelling of Ethereum, and has been reported as incorrect by other translators in these languages multiple times.

### CODE REVIEW

File: src/content/translations/zh/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 1]
```diff
@@ -62,8 +62,13 @@ sourceUrl: https://docs.alchemyapi.io/tutorials/sending-transactions-using-web3-
 **注：**本指南假设您已有一个 Alchemy 帐户、一个以太坊地址或 Metamask 钱包，并已安装 NodeJs 和 npm。 如果没有，按以下步骤操作：
 
 1.  [创建一个免费的 Alchemy 帐户](https://dashboard.alchemyapi.io/signup/)
+<<<<<<< Updated upstream
 2.  [创建 Metamask 帐户](https://metamask.io/)（或获取一个 Etherum 地址）
 3.  [按照这些步骤安装 NodeJs 和 NPM](https://docs.alchemy.com/alchemy/guides/alchemy-for-macs)
+=======
+2.  [创建 Metamask 帐户](https://metamask.io/)（或获取一个 Ethereum 地址）
+3.  [按照这些步骤安装 NodeJs 和 NPM](https://medium.com/guides/alchemy-for-macs)
+>>>>>>> Stashed changes
```
Comment (wackerow) [inline src/content/translations/zh/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:71]:
```suggestion
2.  [创建 Metamask 帐户](https://metamask.io/)（或获取一个 Ethereum 地址）
3.  [按照这些步骤安装 NodeJs 和 NPM](https://medium.com/guides/alchemy-for-macs)
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Looks good @lukassim, just noticed one spot where there were stashed changed that needed to be resolved. Left a code suggestion to correct it, and should be good other than that

**Comment by lukassim** [comment]
Great, thanks @wackerow!

**Review by minimalsm — APPROVED** [review]
LGTM! Let's hold off merging until the Eth2 rebrand is in as it'll introduce more conflicts.

**Comment by minimalsm** [comment]
Merge conflict hell on this 😢

Would it maybe be quicker to throw up a fresh PR for this? @wackerow @corwintines

**Review by wackerow — APPROVED** [review]
Thanks @lukassim !

**Pull request merged**